packages/components/src/local/coa-status-board/helpers/gen-data.tsx

```diff
@@ -209,7 +209,7 @@ export const genData = (
     }
 
     const row: Row = {
-      id: message.message.Reference || message._id,
+      id: message._id || message.message.Reference,
```

Comment:
@lebaphi - do you know the reason for the above change?

All PouchDB documents have an `_id` field, so the `Reference` will never get used.  I _think_ the previous way was the correct way around.

Reply:
Ah I get it from the old code base, but the same with you, I think we should use `message._id` instead of the `message.message.Reference`.
![image](https://user-images.githubusercontent.com/26705621/144404703-f61437dc-25ee-43ef-aa49-b7186020606e.png)

I change it to make the row id is the same with the message id, then we can filter the opened message in the `ExpandedComponent` component.